### PR TITLE
LPS-187175 Enable Form Container in display pages

### PR DESCRIPTION
**Pull request by veroglez**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by veroglez** [comment]
ci:test:sf

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page_structure/components/item_configuration_panels/FormMappingOptions.js [hunk 1]
```diff
@@ -33,6 +35,25 @@ export default function FormMappingOptions({
...
+					label: sub(
+						Liferay.Language.get('x-default'),
+						config.selectedMappingTypes.type.label
+					),
+				};
+			}
+
+			return formType;
+		});
+	}
```
Comment (victorg1991) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page_structure/components/item_configuration_panels/FormMappingOptions.js:55]: For now I think it is fine, but @ealonso we can probably move this to the backend to https://github.com/liferay/liferay-portal/blob/c7ef6cda2c662c88c883ce511c52a1db3ecf8626/modules/apps/layout/layout-content-page-editor-web/src/main/java/com/liferay/layout/content/page/editor/web/internal/util/MappingTypesUtil.java#L36

**Comment by victorg1991** [comment]
LGTM :)

**Comment by victorg1991** [comment]
ci:forward

**Comment by victorg1991** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward

**Comment by victorg1991** [comment]
ci:forward

**Comment by veroglez** [comment]
ci:stop

**Comment by veroglez** [comment]
ci:stop

**Comment by ealonso** [comment]
ci:forward

**Pull request closed without merging**